crates/eframe/src/native/run.rs

```diff
@@ -160,94 +190,122 @@ fn run_and_return(
             // Platform-dependent event handlers to workaround a winit bug
             // See: https://github.com/rust-windowing/winit/issues/987
             // See: https://github.com/rust-windowing/winit/issues/1619
-            #[cfg(target_os = "windows")]
-            winit::event::Event::RedrawEventsCleared => {
-                next_repaint_time = extremely_far_future();
-                winit_app.run_ui_and_paint()
-            }
-            #[cfg(not(target_os = "windows"))]
-            winit::event::Event::RedrawRequested(_) => {
-                next_repaint_time = extremely_far_future();
-                winit_app.run_ui_and_paint()
+            // #[cfg(target_os = "windows")]
+            // winit::event::Event::RedrawEventsCleared => {
+            // windows_next_repaint_times.clear();
+            // winit_app.run_ui_and_paint(None)
+            // vec![EventResult::Wait]
+            // }
+            // #[cfg(not(target_os = "windows"))]
```

Comment:
The problem is that for the windows workaround to work on RedrawEventsCleared i need to render all the viewports and in here i don't know all the window_ids of all the viewports.
I can do that if you want!

But other problem will be the performance, because if we interact with any viewport all viewports will be rendered!

On wine works!
But only if the winit control flow is set to poll when a redraw is requested, but when a redraw is requested from egui will always be set to pull at line 292

From what i seen from @stefnotch is currently working but i will test and on my friend computer!
I will add a new comment when i test this on Windows 10